BearStudio/start-ui-native
# feat: Add color schemes to button component

## Why this PR?

The button didn't have a colorSchema, which prevents to use "primary", "secondary" or other schemes easily.

## What does it contain?

 - Added `colorSchema` prop to the `Button` component
 - Added a screen that shows off the app components, which is available through the dev menu only, and is included only in dev. For the moment, the screen contains only the Button component with it's colorSchema and variants
 - Add the dev menu option to show components, which required to be able to access the navigation globaly (outside a screen)
 - Edited all buttons to use primary colorScheme
 - I also upgraded Magnus version

## Screens
| Screen | Screen | Screen |Screen |
| ---------|------------|-----------|-----------|
| ![Screenshot_20210728_173635_com startuinativeapp](https://user-images.githubusercontent.com/27200095/127352179-b4126d65-385b-4b9f-bfc3-21494f5edd57.jpg)|![Screenshot_20210728_171457_com startuinativeapp](https://user-images.githubusercontent.com/27200095/127348381-a63613ae-f5be-4b33-8eea-54db50fdac14.jpg) |![Screenshot_20210728_170841_com startuinativeapp](https://user-images.githubusercontent.com/27200095/127348383-92a66662-47e0-4d04-889d-7a97bec6af6d.jpg)|![Screenshot_20210728_170834_com startuinativeapp](https://user-images.githubusercontent.com/27200095/127348386-e5554266-9ccd-4ba4-8180-c79069a4ef4e.jpg)





## How to test
 - Ensure that all the Buttons still have the same style
 - Go to the components page using the dev menu
 - Build and ensure the application doesn't break for any reason
